src/app/pages/home/home.component.html

```diff
@@ -60,16 +62,24 @@ <h2 class="mb-3 px-5 home-text header-text">What is virtrolio?</h2>
     <div class="row">
       <div class="order-md-1 col-12 col-sm-12 col-md-12 col-lg-6 col-xl-8 px-5">
         <div class="card mr-4 mb-5 rounded border-0">
-          <h3 class="body-header-text">Create a virtrolio</h3>
+          <h3 class="body-header-text">Why use virtrolio?</h3>
+          <p class="lead body-text">Virtrolio offers you an authentic yearbook signing experience, and that means
+            <em>privacy</em>.
+            You can only view virtrolio messages sent directly to you (see our Privacy Policy for more details).</p>
+        </div>
+        <div class="card mr-4 mb-5 rounded border-0">
+          <h3 class="body-header-text">How can I make a virtrolio?</h3>
           <p class="lead home-text body-text">It's easy to create a virtrolio! Sign in with your Google account
             and a custom virtrolio link will be generated for you. Share this link with your friends so they can sign
             your
             virtrolio.</p>
         </div>
         <div class="card mr-4 rounded border-0">
-          <h3 class="body-header-text">Sign a virtrolio</h3>
-          <p class="lead home-text body-text">Truly express yourself with an array of message customization tools,
-            including colour pickers, fonts, emojis, and a markdown editor.</p>
+          <h3 class="body-header-text">What does signing a virtrolio look like?</h3>
+          <p class="lead home-text body-text">See our interactive example below! We offer you an array of message
+            customization tools,
+            including colour pickers, fonts, emojis, and a markdown editor—all so that you can truly express
```

Comment:
```suggestion
            including <span style="color: #d1155a">colour pickers</span>, <b><i>fonts</i></b>, emojis &#128525;, and a markdown editor—all so that you can truly express
```

Give the description some character 😄